torch/_dynamo/variables/constant.py

```diff
@@ -37,18 +36,29 @@
 class ConstantVariable(VariableTracker):
     @staticmethod
     def create(value, **kwargs) -> VariableTracker:
+        """
+        Create a `ConstantVariable` based on the given value.
+
+        NOTE: the caller must install the proper guards if needed; most often
+        the guard will be `CONSTANT_MATCH`.
```

Comment:
Existing use cases of this (that has a source) all have `CONSTANT_MATCH` installed explicitly, except for 1 case which I fixed in this patch.